tst/EnergyPlus/unit/SingleDuct.unit.cc

```diff
@@ -0,0 +1,404 @@
+		"    Zone 1 ADU,            !- Zone Equipment 1 Name",
+		"    1,                       !- Zone Equipment 1 Cooling Sequence",
+		"    1;                       !- Zone Equipment 1 Heating or No-Load Sequence",
+		"ZoneHVAC:AirDistributionUnit,",
+		"    Zone 1 ADU,    !- Name",
+		"    Zone 1 Supply Inlet,     !- Air Distribution Unit Outlet Node Name",
+		"    AirTerminal:SingleDuct:VAV:NoReheat,  !- Air Terminal Object Type",
+		"    Zone 1 VAV No Reheat;           !- Air Terminal Name",
+		"AirTerminal:SingleDuct:VAV:NoReheat,",
+		"    Zone 1 VAV No Reheat,    !- Name",
+		"    AlwaysOff,               !- Availability Schedule Name",
+		"    Zone 1 Supply Inlet,     !- Air Outlet Node Name",
+		"    Zone 1 Zone Equip Inlet, !- Air Inlet Node Name",
+		"    2.0,                     !- Maximum Air Flow Rate {m3/s}",
+		"    Constant,                !- Zone Minimum Air Flow Input Method",
+		"    0.5;                     !- Constant Minimum Air Flow Fraction",
+		"Schedule:Constant,",
+		"    AlwaysOff,               !- Name",
+		"    ,                        !- Schedule Type Limits Name",
+		"    0;                       !- Hourly Value",
+		"Schedule:Constant,",
+		"    AlwaysOn,               !- Name",
+		"    ,                        !- Schedule Type Limits Name",
+		"    1;                       !- Hourly Value",
+
+		} ) ;
+
+		ASSERT_FALSE( process_idf( idf_objects ) );
+
+		DataGlobals::NumOfTimeStepInHour = 1; // must initialize this to get schedules initialized
+		DataGlobals::MinutesPerTimeStep = 60; // must initialize this to get schedules initialized
+		ScheduleManager::ProcessScheduleInput(); // read schedules
+		ScheduleManager::ScheduleInputProcessed = true;
+		DataEnvironment::Month = 1;
+		DataEnvironment::DayOfMonth = 21;
+		DataGlobals::HourOfDay = 1;
+		DataGlobals::TimeStep = 1;
+		DataEnvironment::DSTIndicator = 0;
+		DataEnvironment::DayOfWeek = 2;
+		DataEnvironment::HolidayIndex = 0;
+		DataEnvironment::DayOfYear_Schedule = General::JulianDay(DataEnvironment::Month, DataEnvironment::DayOfMonth, 1);
+		DataEnvironment::StdRhoAir = Psychrometrics::PsyRhoAirFnPbTdbW( 101325.0, 20.0, 0.0 ); 
+		ScheduleManager::UpdateScheduleValues();
+
+		bool ErrorsFound = false;
+		HeatBalanceManager::GetZoneData(ErrorsFound);
+		ASSERT_FALSE(ErrorsFound);
+		DataZoneEquipment::GetZoneEquipmentData1();
+		ZoneAirLoopEquipmentManager::GetZoneAirLoopEquipment();
+		SingleDuct::GetSysInput();
+		EXPECT_TRUE( compare_err_stream( "" ) );
+		DataHeatBalFanSys::TempControlType.allocate( 1 );
+		DataHeatBalFanSys::TempControlType( 1 ) = DataHVACGlobals::DualSetPointWithDeadBand;
+
+		// node number table
+		//  1   Zone 1 Air Node
+		//  2   Zone 1 Return Node
+		//  3   Zone 1 Supply Inlet
+		//  4   Zone 1 Zone Equip Inlet
+
+		DataZoneEnergyDemands::ZoneSysEnergyDemand.allocate(1);
+		DataZoneEnergyDemands::ZoneSysEnergyDemand( 1 ).RemainingOutputRequired = -2000.0; // Zone 2 has a cooling load
```

Comment:
// Zone 1 has a cooling load...but I don't really care about the innaccurate comment